basic/ncr.py

```diff
@@ -0,0 +1,16 @@
+def nCr(n, r):
+    return (fact(n) / (fact(r)* fact(n - r)))
```

Comment:
```suggestion
    return fact(n) / (fact(r) * fact(n - r))
```

nit: outer parens redundant
nit: "always have the same amount of whitespace on both sides of a binary operator"